codecov.yml

```diff
@@ -4,3 +4,5 @@ coverage:
   status:
     project: no
     patch: yes
+fixes:
+  - "::src/"
```

Comment:
If I pick a [previous PR](https://app.codecov.io/gh/mrc-ide/hint/compare/471) on codecov I can't actually see annotated files under the "Coverage Changes" or "Files" tabs. This is because we run coverage from inside the `src` folder, but codecov needs to know that that the path on GitHub does start with `src`.

But I think this needs more thought (and comparison with other projects) so I'll remove it here and create a maintenance ticket.

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2234